src/modal.rs

```diff
@@ -41,11 +41,12 @@ async fn execute<U: Send + Sync, E, M: Modal>(
     defaults: Option<M>,
 ) -> Result<M, serenity::Error> {
     let interaction = ctx.interaction.unwrap();
+    let interaction_id = interaction.id.to_string();
 
     // Send modal
     interaction
         .create_interaction_response(ctx.discord, |b| {
-            *b = M::create(defaults);
+            *b = M::create(defaults, Some(interaction_id.clone()));
```

Comment:
Can't you just pass `interaction.id.to_string()` directly in here?

Reply:
Yeah, I wasn't sure whether to do that or just re-use the existing variable I made. I've changed it to use `interaction.id.to_string()` though.

Reply:
Oh, I didn't realize `interaction.id.to_string()` was used later again. Then it makes total sense to use a variable instead of inlining. I'll quickly change it back locally so you don't have to go through the trouble